src/components/Menu-Badge/MenuBadge.tsx

```diff
@@ -0,0 +1,100 @@
+import { Box, Typography } from '@mui/material';
+
+import { COLOR } from './MenuBadge.constraints';
+
+type MenuBadgeProps = {
+  label: string;
+  variant?: 'default' | 'color';
+};
+
+export const MenuBadge: React.FC<MenuBadgeProps> = ({ label, variant = 'default' }) => {
```

Comment:
Using `React.FC` is discouraged in some codebases because it implicitly adds children and may cause inference issues. If you keep it, ensure you imported React; otherwise prefer explicit function signature with typed props.

**Sugerencia:**
```typescript
Prefer: `export const MenuBadge = ({ label, variant = 'default' }: MenuBadgeProps): JSX.Element => { ... }`
```